core/src/main/java/org/elasticsearch/action/DocWriteResponse.java

```diff
@@ -314,8 +307,8 @@ public XContentBuilder innerToXContent(XContentBuilder builder, Params params) t
                 .field(_ID, id)
                 .field(_VERSION, version)
                 .field(RESULT, getResult().getLowercase());
-        if (getTookInNanos() >= 0) {
-            builder.field(TOOK, getTookInMillis()); // Building 'took' in milliseconds for readability
+        if (getTook().nanos() >= 0) {
+            builder.field(TOOK, getTook().millis());    // Building 'took' in milliseconds for readability
```

Comment:
Keep it in the current form.

`builder.timeValueField` is useful because if you specify `?human` then it'll output something like `1.7s` which is useful, but we never seem to do it for `took`.